soymsg/pomsg/pomsg_test.go

```diff
@@ -143,3 +148,46 @@ func TestNewMessage(t *testing.T) {
 		}
 	}
 }
+
+func TestFallbackBundle(t *testing.T) {
```

Comment:
This is sort of a lot of code to copy/paste. Maybe a loop over "en", "en_UK" is preferable in the other test?

Reply:
Done